consensus/state.go

```diff
@@ -520,6 +520,14 @@ func (cs *State) updateHeight(height int64) {
 }
 
 func (cs *State) updateRoundStep(round int32, step cstypes.RoundStepType) {
+	if !cs.replayMode {
+		if round != cs.Round || round == 0 && step == cstypes.RoundStepNewRound {
+			cs.metrics.MarkRound(cs.Round, cs.StartTime)
```

Comment:
Is `cs.StartTime` the right time value to be comparing with? Isn't this only set when we update state to the next height and not after every round?

Also in `updateToState`, `cs.StartTime` is set _after_ `updateRoundStep` is called

Reply:
Removed `RoundDuration` metric.